.readthedocs.yaml

```diff
@@ -17,6 +17,6 @@ sphinx:
 formats:
   - pdf
 
-python:
-  install:
-    - requirements: help/requirements.txt
+    # python:
```

Comment:
Do we need to keep this commented bit around?

Reply:
It actually needs to be uncommented. I thought myst_parser was included on RTD, but nah. :(

Reply:
8bd3e09